index.html

```diff
@@ -133,5 +144,47 @@ <h2>About Me</h2>
     </div>
     </section>
     <br>
+    <section class="footer">
+    <h2 class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in <span class="toll">touch.</span></h2>
+    
+    <form  class="player" action="https://formspree.io/f/xayagqrn" method="post">
+    <ul class="name">
+    <li class="form">
+    <label for="fullName">
+    <input type="text" name="full-name" placeholder="  Full name" maxlength="30" id="fullName" required>
+        </label></li>
+        
+    <li class="form">
+    <label for="email">
+        <input type="email" name="email" id="email" placeholder="  Email address" required>
+        </label></li>
+        
+        <li class="textarea">
+        <label for="message">
+        <textarea name="message" id="message" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
+        </label></li>
+    
+    <li class="buttonclub">
+    <button type="submit" class="button_class">Get in touch</button>
+    </li>
+    </ul>
+</form>
+
+    <div class="border"><img src="img_line-line.png" alt=""></div>
+
+    <ul class="icongroup">
+        <li class="githubx" ><a href="#"><img src="pictures/githreal.svg" alt="Github"></a></li>
+
+        <li class="linkedinx" ><a href="#"><img  src="Pictures/linkedreal.svg" alt="linkedin"></a></li>
+
+        <li class="angelistx"><a href="#"><img src="Pictures/angelist.svg" alt="angel.co"></a></li>
+
+        <li class="twitterx"><a href="#"><img src="Pictures/twittereal.svg" alt="twitter"></a></li>
+
+        <li class="mediumx"><a href="#"><img src="Pictures/mediumreal.svg" alt="medium.com"></a></li>
```

Comment:
- [ ] These links are broken, probably because the address is wrong, if you replace `Pictures` to `pictures` the `svg`s will work 
![image](https://user-images.githubusercontent.com/77038610/145394521-3ca85b15-a7e7-477f-b081-839011692eac.png)

- [ ]  In order to meet the requirements, please do not forget to center the links as shown in Figma😉

Reply:
I have made these changes to my code now. I used small letter "p" to call the icons from the asset folder named /pictures. I was also able to change each of the icons to single icon.